GoogleSignIn/Sources/GIDAuthorizationResponse/API/GIDAuthorizationResponseHandling.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+@class GIDAuthFlow;
+@class GIDConfiguration;
+@class OIDAuthorizationResponse;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// A list of potential current flow names.
+typedef NS_ENUM(NSInteger, GIDFlowName) {
+/// The Sign In flow.
+  GIDFlowNameSignIn = 0,
+/// The Verify flow.
+  GIDFlowNameVerify = 1,
```

Comment:
Can we rename this to `GIDFlowNameVerifyAccountDetail`?

Reply:
Done.